src/components/v1/V1Project/Paid.tsx

```diff
@@ -179,15 +177,7 @@ export default function Paid() {
             marginLeft: 10,
           }}
         >
-          {currentFC.currency.eq(CURRENCY_USD) ? (
-            <span style={secondaryTextStyle}>
-              <CurrencySymbol currency={CURRENCY_ETH} />
-              {formatWad(balance, { precision: 2, padEnd: true })}{' '}
-            </span>
-          ) : (
-            ''
-          )}
-          {formatCurrencyAmount(balanceInCurrency)}
+          <ETHAmount amount={balance} />
```

Comment:
@johnnyd-eth do we still need this conditional logic here from previously?

Reply:
Yep I thought it might be worth removing it after adding these tooltips, but on second thought I think it's best left as it is